packages/framework/src/Model/Order/Item/OrderItemFacade.php

```diff
@@ -42,9 +42,9 @@ class OrderItemFacade
     protected $orderPriceCalculation;
 
     /**
-     * @var \Shopsys\FrameworkBundle\Model\Order\Item\OrderProductFactoryInterface
+     * @var \Shopsys\FrameworkBundle\Model\Order\Item\OrderItemFactoryInterface
      */
-    protected $orderProductFactory;
+    private $orderItemFactory;
```

Comment:
@grossmannmartin We use `protected` for properties in Facades for easier extendability